### DISCUSSION TITLE
temp dir question

### DISCUSSION THREAD

**Discussion by Thedeadlyart** [swar/Swar-Chia-Plot-Manager#1020] [Q&A] [answered]
i have a pc with 500gb ssd every once and a while i have a issue where it runs out of room trying to do 2 parallel plots back to back even with delay sometimes it runs fine just not always. i was trying to do the setup in the second section that shows like this:

    temporary_directory: D:\chiatemp
      - C:\chiatemp
my issue is i can't add that to the first job list it just won't start a job period but if i remove the - C stuff it runs the job. i was wondering if there is a issue where that function doesn't work anymore. i wanted to have that so if that issue come up instead of it stopping the pot it will just have the little extra that wouldn't fit go on another drive like what the description says it is susppose to do. i tried setting it up as it was single plot first job and single on second job with the - C but second won't start.

**Comment by cherymann** [comment, marked as answer]
Try this

> temporary_directory: 
>     \- D:\chiatemp
>   - C:\chiatemp

**Reply by Thedeadlyart** [reply]
ty that worked i didn't even think to try that.

**Reply by cherymann** [reply]
Anytime! Have fun plotting!
